### PR TITLE
SSR-2360 User guide for Custom CSS

### PR DESCRIPTION
**Pull request by voduchau**

### CODE REVIEW

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
The next version will be v1.5.0, so please change to that.

File: docs/document/docs/getting-started/custom-css.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+id: custom-css
+title: Custom CSS
+sidebar_label: Custom CSS
+---
+
+## Overview
+
+**Kintone UI Component** provide some Custom CSS properties for each component by using [CSS inheritance](#css-inheritance) and [CSS variables and custom properties](#css-variables-and-custom-properties) together, it's easy to create themable elements. By applying css selectors to customize CSS custom properties, tree-based and per-instance theming is straightforward to apply.
```
[the remark this answers is not in the document]
Comment (voduchau) [reply docs/document/docs/getting-started/custom-css.md:9]: themable element meaning an element that we can customize its css.
I will change "themable" to "customizable".

File: docs/document/docs/getting-started/custom-css.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### CSS inheritance
+
+CSS inheritance lets parent and host elements propagate certain CSS properties to their descendants.
+
+### CSS variables and custom properties
+
+All CSS custom properties (`--custom-property-name`) inherit. You can use this to make your component's styles configurable from outside.
+
+KUC will publicize some properties for each element, based on that users can customize css easily (_For now we will publicize some properties for **Button** and **Text** component first_).
```
[the remark this answers is not in the document]
Comment (voduchau) [reply docs/document/docs/getting-started/custom-css.md:19]: >What is the meaning of "publicize" in this case?
I change "publicize" to "publish".

>I think it is good not to mention the component names.
Ok, I remove it

File: docs/document/docs/getting-started/custom-css.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (voduchau) [reply docs/document/docs/getting-started/custom-css.md]: I also change to v1.5.0.
Thank you

[the remark this answers is not in the document]
Comment (TomokoMiyake) [reply docs/document/docs/getting-started/custom-css.md]: Got it, thank you!

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
Please update docs/document/components.

File: docs/document/docs/getting-started/custom-css.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+<style>
+  my-element {
+    --my-background: #d9092f;
+  }
+  my-element.sample-class {
+    --my-background: #a8a632;
+  }
+   my-element#sample-id {
+    --my-background: #1ac9b8;
+  }
```
Comment (TomokoMiyake) [inline docs/document/docs/getting-started/custom-css.md:46]: You are using "my-element" before class name/id here.
And on Sample Code of Text, you are directly using class name/id as element selector.
I think it's better to unify it.

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]

**Review by TomokoMiyake — APPROVED** [review]
LGTM!

File: docs/document/docs/getting-started/custom-css.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+id: custom-css
+title: Custom CSS
+sidebar_label: Custom CSS
+---
+
+## Overview
+
+**Kintone UI Component (KUC)** provides some Custom CSS properties for each component by using [CSS inheritance](#css-inheritance) and [CSS variables and custom properties](#css-variables-and-custom-properties) together. It's easy to create customizable elements. By applying CSS selectors to customize CSS custom properties, tree-based and per-instance theming are straightforward to apply.
```
Comment (huuchi1778) [inline docs/document/docs/getting-started/custom-css.md:9]:
```suggestion
**Kintone UI Component (KUC)** provides some Custom CSS properties for each component by using [CSS inheritance](#css-inheritance), [CSS variables and custom properties](#css-variables-and-custom-properties) together. It's easy to create customizable elements by applying CSS selectors to customize CSS custom properties. Tree-based and per-instance theming are straightforward to apply.
```

File: docs/document/docs/getting-started/custom-css.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+### CSS inheritance
+
+CSS inheritance lets parent and host elements propagate certain CSS properties to their descendants.
+
+### CSS variables and custom properties
+
+All CSS custom properties (`--custom-property-name`) inherit. You can use this to make your component's styles configurable from outside.
```
Comment (huuchi1778) [inline docs/document/docs/getting-started/custom-css.md:17]:
```suggestion
All CSS custom properties (`--custom-property-name`) inherit their value from their parent. You can use this to make your component's styles configurable from the outside.
```
How about this to make the sentence clearer?

File: docs/document/docs/getting-started/custom-css.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+KUC will publish some properties for each element, based on that users can customize CSS easily.
```
Comment (huuchi1778) [inline docs/document/docs/getting-started/custom-css.md:19]:
```suggestion
KUC will publish some properties for each element and based on that, users can easily customize CSS properties of the element.
```

File: docs/document/docs/getting-started/custom-css.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+## Usage
+
+You can set the value for `--my-background` by using the CSS element selector:
```
Comment (huuchi1778) [inline docs/document/docs/getting-started/custom-css.md:23]:
```suggestion
You can set the value for `--my-background` by using the CSS selectors:
```
It seems like you can use not only element selectors but other CSS selectors in general.

File: docs/document/docs/getting-started/custom-css.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+  #sample-id {
+    --my-background: #1ac9b8;
+  }
+</style>
+<my-element></my-element>
+<my-element class="sample-class"></my-element>
+<my-element id="sample-id"></my-element>
+```
+
+> You can find the list of Custom CSS properties in the **Custom CSS** section on each component page.
```
Comment (huuchi1778) [inline docs/document/docs/getting-started/custom-css.md:53]:
```suggestion
> You can find a list of the Custom CSS properties in the **Custom CSS** section on each component page.
```

**Review by huuchi1778 — COMMENTED** [review]
Please have a look at my suggestions.
Thank you!

**Pull request merged**